packages/sdks/src/components/render-block/block-styles.lite.tsx

```diff
@@ -0,0 +1,39 @@
+import { BuilderBlock } from '../../types/builder-block';
+import RenderInlinedStyles from '../render-inlined-styles.lite';
+import { useState } from '@builder.io/mitosis';
+
+export type BlockStylesProps = {
+  block: BuilderBlock;
+};
+
+export default function BlockStyles(props: BlockStylesProps) {
+  const state = useState({
+    camelToKebabCase(string: string) {
+      return string
+        .replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2')
+        .toLowerCase();
+    },
+
+    get css(): string {
+      // TODO: media queries
```

Comment:
Is this a limitation for all the new SDKs, as in do they not support the different device size styles in the Builder editor?

Reply:
TBH I'm unsure. I copy-pasted this code from here:

https://github.com/BuilderIO/builder/blob/c9b5391bd7edcc17e0b25b03fef13f5c05fc9526/packages/sdks/overrides/vue/src/components/block-styles.vue#L12-L31

It was a Vue-specific override, and I extracted it out to be usable by other frameworks as well.